test/unit/model/upsert.test.js

```diff
@@ -14,6 +14,11 @@ describe(Support.getTestDialectTeaser('Model'), () => {
     describe('method upsert', () => {
       before(function () {
         this.User = current.define('User', {
+          id: {
```

Comment:
this is redundant, an auto-incremented integer primary key is added by default if the model doesn't define its own and doesn't use the "noPrimaryKey" option